### PR TITLE
docs: updates and improve README.md

### PR DESCRIPTION
**Pull request by LucasPerso**
It was about time we had a nice Readme !

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,87 +1,105 @@
...
 
-Create the `routes.js` file:
+It bundles your application and optimizes its performances. The bundle is also minified.
 
-```js
-import { Route } from 'vitaminjs';
+## Frequently Asked Questions
+We have started to answer some frequently asked questions. You can find all of them here: [FAQ](https://github.com/Evaneos/vitaminjs/blob/master/FAQ.md)  
+If you have any questions you think are worth be displayed, please raise an issue in the project.  
+We will try to update these questions as often as possible.
```
Comment (johangirod) [inline README.md]: No need for this last sentence. It seems a little bit self-degrading :p

File: README.md [hunk 2]
```diff
@@ -1,87 +1,105 @@
...
-    "server": {
-        "layout": "__vitamin__/src/server/components/DivLayout"
-    }
-}
-```
+## Inspiration
+
+[Universal Redux](https://github.com/bdefore/universal-redux)  
+[React-Starter-Kit](https://github.com/kriasoft/react-starter-kit)  
+[Create-React-App](https://github.com/facebookincubator/create-react-app)
```
Comment (johangirod) [inline README.md]: Next.js ?

File: README.md [hunk 3]
```diff
@@ -1,87 +1,105 @@
...
- Build toolchain as a dependency for react/redux application, with a strong emphasis put on DX (Developer eXperience)
-</big></p>
+<p align="center">
+	<img width="125" src="https://upload.wikimedia.org/wikipedia/commons/a/a4/Emojione_1F34D.svg" />
+</p>
 
-## Why ?
-Actual state of development for react apps leads to a tremendous amount of boilerplate code for initializing the tooling. Usually, we proceed by finding (or creating) a boilerplate project with approximatly the stack we need, fork it, and start working on it.
+<big><h1 align="center">VitaminJS</h1></big>
+<p><big>Build framework to start a server-rendered React/Redux application with a strong focus on developer experience
```
Comment (johangirod) [inline README.md]: I'm not 100% sure with this tagline but I can't find any better

File: README.md (hunk 2 above)
Comment (LucasPerso) [inline README.md]: I thought of mentioning it as well but I don't think NextJS really inspired us. On the contrary, CRA really helped us to see what we want to achieve.

Comment (johangirod) [inline README.md]: Ok, seems fair. But it's weird to not see it here. Maybe a `similar project` section would be more expected ?

**Comment by johangirod** [comment]
Anyway, you can merge ASAP, we can make the requested change in a second time, it's not like it was blocking.

**Pull request merged**